docs/policies/code-of-conduct.md

```diff
@@ -6,24 +6,6 @@ Alex’s Lemonade Stand Foundation’s (ALSF) Childhood Cancer Data Lab (Data La
 We are committed to creating a friendly and respectful place for contributing and learning.
 Everyone participating in the Open Single-cell Pediatric Cancer Atlas (OpenScPCA) project is expected to show respect and courtesy to others.
 
-**Table of contents:**
```

Comment:
Did you mean to delete this?

Reply:
I did, but - https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/335#issuecomment-2032619737